### DISCUSSION TITLE
Input Prompts Notebook error

### DISCUSSION THREAD

**Discussion by NoorAhs** [opengeos/segment-geospatial#114] [Q&A] [answered]
![MicrosoftTeams-image](https://github.com/opengeos/segment-geospatial/assets/134938184/b6c40a05-422a-4390-9242-c4f704c45e13)

 It gives this error when I run this one, can you please tell what is the problem?

**Comment by giswqs** [comment]
What changes did you make?

**Comment by NoorAhs** [comment]
I changed nothing

Get Outlook for iOS<https://aka.ms/o0ukef>

Noor Ahsan
Research Assistant / Interns
T:
E: ***@***.***
www.mbzuai.ac.ae<http://www.mbzuai.ac.ae>

Mohamed bin Zayed University of Artificial Intelligence
Masdar City, Building 1B, 3rd Floor
Abu Dhabi, UAE

[cid:32_5f0d4623-c70a-4f6f-aa82-dcb8eb00c458.png]
 The world’s first specialized research-based AI university
[cid:march_email-signature-v2-with-space_127bd5cb-f3b0-4bee-bdf0-39873104f738.gif]<https://mbzuai.ac.ae/sustainability/>


________________________________

**Comment by giswqs** [comment, marked as answer]
It works fine on my end. Try restarting the kernel after installing package.

**Comment by NoorAhs** [comment]
I am running this on collab .  When I select location on map after that it asks for installing localtile server. I used the command "pip install localtileserver"
Then initialize SAM class , after that when I run "Image segmentation with input points" block it gives this error
  "no valid pixel coordinates found"

Get Outlook for iOS<https://aka.ms/o0ukef>

Noor Ahsan
Research Assistant / Interns
T:
E: ***@***.***
www.mbzuai.ac.ae<http://www.mbzuai.ac.ae>

Mohamed bin Zayed University of Artificial Intelligence
Masdar City, Building 1B, 3rd Floor
Abu Dhabi, UAE

[cid:32_5f0d4623-c70a-4f6f-aa82-dcb8eb00c458.png]
 The world’s first specialized research-based AI university
[cid:march_email-signature-v2-with-space_127bd5cb-f3b0-4bee-bdf0-39873104f738.gif]<https://mbzuai.ac.ae/sustainability/>


________________________________

**Comment by giswqs** [comment]
Open the notebook in Google Colab and run all codes without making any changes. If you make changes, make sure you are doing it properly by changing coordinates. Otherwise, it will not work.

**Comment by NoorAhs** [comment]
If I run all then it doesn’t show error
But I need to select a particular area for my work. Can you please tell me   What changes in coordinates are required If I select a particular area?

Get Outlook for iOS<https://aka.ms/o0ukef>

Noor Ahsan
Research Assistant / Interns
T:
E: ***@***.***
www.mbzuai.ac.ae<http://www.mbzuai.ac.ae>

Mohamed bin Zayed University of Artificial Intelligence
Masdar City, Building 1B, 3rd Floor
Abu Dhabi, UAE

[cid:32_5f0d4623-c70a-4f6f-aa82-dcb8eb00c458.png]
 The world’s first specialized research-based AI university
[cid:march_email-signature-v2-with-space_127bd5cb-f3b0-4bee-bdf0-39873104f738.gif]<https://mbzuai.ac.ae/sustainability/>


________________________________
